who-europe-harvard.csl

```diff
@@ -48,30 +51,31 @@
   </macro>
   <macro name="access">
     <choose>
-      <if variable="URL">
-        <group delimiter=", ">
+      <if variable="DOI">
+        <text variable="DOI" prefix="doi:"/>
+      </if>
+      <else-if match="any" variable="URL">
+        <group delimiter=", " prefix="(" suffix=")">
           <text variable="URL"/>
           <group delimiter=" ">
-            <text term="accessed"/>
+            <text value="accessed "/>
```

Comment:
```suggestion
            <text term="accessed"/>
```
should work just fine, no? the space is already in the delimiter. If so, same in the other style